### ISSUE TITLE
Upload

### ISSUE THREAD

**Issue by iboinas** [git-ftp/git-ftp#167]

**Comment by tuffz** [comment]
+1

**Comment by mkllnk** [comment]
Try `-v` for verbose. It displays all set options and doesn't suppress curl's stdout.

I think we should improve the man page here:

       -v, --verbose
              Be verbose.

**Comment by iboinas** [comment]
thank u for feedback!
i tried it, but is only displaying all lines as a bunch of (sharp-symbol)
##################################### 100,0%
at least it blinks (each file a new line) and it does not seem like hanged....
what can be wrong, not to display the file that is at ??

**Comment by mkllnk** [comment]
Ah, okay, you would like to see the file name as well. Sorry, I overlooked that. Okay, we can do that at some point.

**Comment by iboinas** [comment]
Yeah, more explicitly, what i point as "missing" is some kind of progress
indicator...

goes on file 1/1500.... 2/1500... ...... 556/1500 ... and so on until
1500/1500  :)

Just because when uploading conposer vendor folder, it's thousands of
files! :P

**Comment by jarodium** [comment]
+1

**Comment by afranioce** [comment]
+1

**Comment by vcabreupy** [comment]
+1

**Comment by pqrs** [comment]
+1

**Comment by jozefcuryllo** [comment]
+1
